templates/helm.rst.gotmpl

```diff
@@ -0,0 +1,495 @@
+ImagePullSecrets customization
+------------------------------
+
+To provide `imagePullSecrets`` object references, you need to specify them using a following structure:
+
+.. code-block:: bash
+
+   imagePullSecrets:
+     - image-pull-secret1
+     - image-pull-secret2
+
+==========
+NFD labels
+==========
+
+The NFD labels required by the Network Operator and GPU Operator:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Label
+     - Location
+   * - feature.node.kubernetes.io/pci-15b3.present
+     - Nodes containing NVIDIA Networking hardware
+   * - feature.node.kubernetes.io/pci-10de.present
+     - Nodes containing NVIDIA GPU hardware
+
+=======================
+Node Feature Discovery
+=======================
+
+Node Feature Discovery Helm chart customization options can be found `here <https://github.com/kubernetes-sigs/node-feature-discovery/blob/master/deployment/helm/node-feature-discovery/values.yaml>`__. Following is a list of overriden values by NVIDIA Operator Helm Chart:
+
+.. list-table::
+   :header-rows: 1
+
+   {{- define "chart.nfdOpValuesTable" }}
+   * - Name
+     - Type
+     - Default in NVIDIA Network Operator
+     - Notes
+     {{- range .Values }}
+        {{- if hasPrefix "node-feature-discovery" .Key}}
+   * - {{ .Key }}
+     - {{ .Type }}
+     - {{ if .Default }}{{ .Default }}{{ else }}{{ "" }}{{ end }}
+     - {{ if .Description }}{{ .Description }}{{ else }}{{ .AutoDescription }}{{ end }}
+       {{- end }}
+     {{- end }}
+   {{- end }}
+
+{{ template "chart.nfdOpValuesTable" . }}
+
+=======================
+SR-IOV Network Operator
+=======================
+
+SR-IOV Network Operator Helm chart customization options can be found `here <https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/master/deployment/sriov-network-operator/README.md>`__. Following is a list of overriden values by NVIDIA Operator Helm Chart:
+
+.. list-table::
+   :header-rows: 1
+
+   {{- define "chart.sriovValuesTable" }}
+   * - Name
+     - Type
+     - Default in NVIDIA Network Operator
+     - Notes
+     {{- range .Values }}
+        {{- if hasPrefix "sriov-network-operator" .Key}}
+   * - {{ .Key }}
+     - {{ .Type }}
+     - {{ if .Default }}{{ .Default }}{{ else }}{{ "" }}{{ end }}
+     - {{ if .Description }}{{ .Description }}{{ else }}{{ .AutoDescription }}{{ end }}
+       {{- end }}
+     {{- end }}
+   {{- end }}
+
+{{ template "chart.sriovValuesTable" . }}
+
+===================
+Container Resources
+===================
+
+Optional `requests and limits <https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/>`_ can be configured for each container of the sub-resources deployed by the Network Operator by setting the parameter ``containerResources``.
+
+For example:
+
+.. code-block:: yaml
+
+    containerResources:
+    - name: "mofed-container"
+      requests:
+        cpu: "200m"
+        memory: "150Mi"
+      limits:
+        cpu: "300m"
+        memory: "300Mi"
```

Comment:
I believe this is not true anymore... Those fields are commented in the `helm chart`.